packages/as-types/assembly/serialization/numbers.ts

```diff
@@ -1,7 +1,12 @@
 /**
```

Comment:
It would be nice to include a short introductory comment at the beginning to provide a brief overview of the module's purpose.